docs/excel/custom-functions-naming.md

```diff
@@ -1,75 +1,101 @@
+A function's `name` and `id` share some similar rules.
 
-- Both function `name` and `id` must start with a letter and have a minimum limit of three characters.
+- Both must start with a letter and be at least three characters.
+- `id`: Only A–Z, 0–9, underscore, and period characters allowed.
+- `name`: Any Unicode letters, underscore, and period characters allowed.
 
-Excel uses uppercase letters for built-in function names (such as `SUM`). Use uppercase letters for your custom function's `name` and `id` as a best practice.
+Excel shows built‑in function names in uppercase letters (such as `SUM`). Using uppercase helps your custom functions fit in.
 
-A function's `name` shouldn't be the same as:
+Avoid names that match:
 
-- Any cells between A1 to XFD1048576 or any cells between R1C1 to R1048576C16384.
-
-- Any Excel 4.0 Macro Function (such as `RUN`, `ECHO`).  For a full list of these functions, see [this Excel Macro Functions Reference document](https://www.myonlinetraininghub.com/cdn/files/Excel%204.0%20Macro%20Functions%20Reference.pdf).
+- A cell reference (A1 to XFD1048576 or R1C1 style).
+- An Excel 4.0 Macro Function (such as `RUN`, `ECHO`). For a full list of these functions, see [this Excel Macro Functions Reference document](https://www.myonlinetraininghub.com/cdn/files/Excel%204.0%20Macro%20Functions%20Reference.pdf).
 
 ## Naming conflicts
 
-If your function `name` is the same as a function `name` in an add-in that already exists, the **#REF!** error will appear in your workbook.
+If your function `name` conflicts with one from another add‑in, Excel shows the **#REF!** error.
 
-To fix a naming conflict, change the `name` in your add-in and try the function again. You can also uninstall the add-in with the conflicting name. Or, if you're testing your add-in in different environments, try using a different namespace to differentiate your function (such as `NAMESPACE_NAMEOFFUNCTION`).
+Fix conflicts by renaming your function or uninstalling the other add‑in. For testing in multiple environments, use a short namespace prefix (such as `ACME_FUNCTIONNAME`).
 
 ## Best practices
 
-- Consider adding multiple arguments to a function rather than creating multiple functions with the same or similar names.
-- Avoid ambiguous abbreviations in function names. Clarity is more important than brevity. Choose a name like `=INCREASETIME` rather than `=INC`.
-- Function names should indicate the action of the function, such as =GETZIPCODE instead of ZIPCODE.
-- Consistently use the same verbs for functions which perform similar actions. For example, use `=DELETEZIPCODE` and `=DELETEADDRESS`, rather than `=DELETEZIPCODE` and `=REMOVEADDRESS`.
-- When naming a streaming function, consider adding a note to that effect in the description of the function or adding `STREAM` to the end of the function's name.
+- Prefer extra function arguments over multiple near-duplicate function names.
```

Comment:
The original needed a better explanation, and it would be good to show a contrasting sample here like with the others.

Reply:
I added an example. Let me know what you think of it.